packages/installation-delta-action/src/components/Header/Header.js

```diff
@@ -0,0 +1,68 @@
+import React, {useEffect, useState} from 'react'
+import PropTypes from 'prop-types'
+import {Typography} from 'tocco-ui'
+import styled from 'styled-components'
+
+const DICTIONARY = {
+  older: 'Älter',
+  newer: 'Neuer'
+}
+
+const StyledHeader = styled.div`
+  margin-bottom: 8px;
+`
+const Header = ({keys, delta}) => {
+  const [labelInstance, setLabelInstance] = useState('')
+  const [statusInstance, setStatusInstance] = useState('')
+  const [labelDelta, setLabelDelta] = useState('')
+  const [statusDelta, setStatusDelta] = useState('')
+
+  const getFetchRequest = async key => {
+    const options = {credentials: 'include'}
+    const url = `${__BACKEND_URL__}/nice2/rest/entities/Installation/${key}?relations=!&_fields=!`
+    const response = await fetch(url, options)
+    return response.json()
+  }
+
+  const fetchLabels = async() => {
+    const [instanceInstallation, deltaInstallation] = await Promise.all(
+      [getFetchRequest(keys[0]),
+        getFetchRequest(keys[1])]
```

Comment:
maybe throw meaningful error if `keys` does not contain 2 keys or use custom proptype function for validation - or at least document it in readme.md

Reply:
Thought about it before but this check happens in nice. imo not worth the effort. But i will add it to the readme.